packages/automatic-releases/src/main.ts

```diff
@@ -306,10 +316,11 @@ export const main = async (): Promise<void> => {
       owner: context.repo.owner,
       repo: context.repo.repo,
       tag_name: releaseTag,
-      name: args.releaseTitle ? args.releaseTitle : releaseTag,
+      ...(args.autoGenerateReleaseNotes ? {} : {name: args.releaseTitle ? args.releaseTitle : releaseTag}),
```

Comment:
@lauravuo  i tried using this via your forked action.yml https://github.com/lauravuo/action-automatic-releases/blob/test-changes/action.yml for some reason the custom `title` is not getting reflected when i generate a release (it uses the git tag as the release title)

im using the action this way 

```
      - uses: "lauravuo/action-automatic-releases@test-changes"
        with:
          repo_token: ${{ secrets.OPS_GHA_USER_TOKEN }}
          draft: false
          title: "Dippr release - ${{ steps.date.outputs.date }}"
          automatic_release_tag: ${{ steps.get-latest-tag.outputs.tag }}
          prerelease: false
          auto_generate_release_notes: true
```